sections/game_modes.tex

```diff
@@ -11,7 +11,7 @@ \subsection*{Clash}
 
 \subsection*{Campaign}
 A single player mode of interconnected Scenarios against an enemy AI.
-For rules unique to Solo Mode\index{Solo Mode}, see \hyperlink{AIrules}{the AI rules}.
+For rules unique to Solo Mode\index{Solo Mode}, see \hyperlink{AIrules}{the AI rules} and \hyperlink{End}{Scenario End}.
```

Comment:
```suggestion
For rules unique to Solo Mode\index{Solo Mode}, see \hyperlink{AIrules}{the AI rules} and \hyperlink{End}{Scenario end}.
```
It makes sense to capitalise everything in the titles, but I think "end" can be lowercase.